enterprise/frontend/src/metabase-enterprise/whitelabel/components/LoginPageIllustrationDescription.tsx

```diff
@@ -0,0 +1,32 @@
+import { t } from "ttag";
+
+import { Flex, Icon, Stack, Text, HoverCard } from "metabase/ui";
+
+interface LoginPageIllustrationDescriptionProps {
+  errorMessageContainerId: string;
+}
+export function LoginPageIllustrationDescription({
+  errorMessageContainerId,
+}: LoginPageIllustrationDescriptionProps) {
+  return (
+    <Text fw="bold" transform="none">
+      <Flex align="center">
+        {t`Login page`}
+        <HoverCard position="top-start">
+          <HoverCard.Target>
+            <Icon aria-hidden name="info" />
```

Comment:
in what way exactly? Width or preview border or the select width?
1. Setting width: I think I made it to almost match the existing Metabot toggle.
2. Preview image border: Maybe I could add a bit of vertical padding to the preview image, so it doesn't feel too tight in there.
3. Select width: It's only narrower when the custom option is selected to accommodate for the file name or the `Remove uploaded image` description (which will displays when you navigate away and back to this setting again since we don't store the image name anyway, we couldn't display the image name except only when users are uploading them)

Reply:
I'll see what I can do. Thanks for the clarification.

Reply:
Welp, forgot to do this I'll include it in the next PR.